chart/kubeapps/values.yaml

```diff
@@ -1606,6 +1606,7 @@ kubeappsapis:
   ##
   enabledPlugins:
     - helm
+    - kapp-controller
```

Comment:
Did you intend to enable the kapp-controller plugin by default? I wouldn't have thought to do this, especially in this first release, but even in others when users need to install kapp controller etc. It may not be an issue for the kapp-controller plugin, as it may not try to query any carvel resources until a request comes in, but the flux plugin will correctly cause the kubeappsapis server to fail to start if it doesn't find the required resources.

Reply:
Hah, I think you saw this yourself before I submitted and removed. Sorry for the noise.